app/views/govuk_publishing_components/components/docs/summary_list.yml

```diff
@@ -53,6 +53,13 @@ examples:
       - field: "Body"
         value: "After the government decided in 2013 to expand the remit of the Committee to include public service providers, the Committee on Standards in Public Life produced our first report on the issue: Ethical Standards for Providers of Public Services in 2014."
 
+  with_custom_link_on_section:
+    data:
+      <<: *default-example-data
+      edit:
+        href: "custom-link"
+        link_text: "Reorder"
```

Comment:
I'd consider using either different example link text or a different example in the summary list items, as it looks odd having a "Reorder" link for a "Title, summary and body".

Reply:
Done 👍